src/ebook.cms.tsx

```diff
@@ -38,7 +38,7 @@ export const EbookCmsEditorComponent = {
     return {
       sectionTitle: match[1],
       ebookName: match[2],
-      ebookDescription: match[3],
+      ebookDescription: match[3].replaceAll('&#39;', '\''),
```

Comment:
Hmmm, I suppose that we could approach this in a more generic React friendly way 🤔

<img width="1684" alt="Screenshot 2024-02-21 at 14 08 35" src="https://github.com/bright/new-www/assets/357191/17c27267-1835-4838-b77a-f67a663e7cd8">


I you find that this approach is easier feel free to use it. If let's merge this version.